src/consensus/consensus.h

```diff
@@ -16,6 +16,8 @@ static const unsigned int MAX_BLOCK_WEIGHT = 4000000;
 static const unsigned int MAX_BLOCK_BASE_SIZE = 1000000;
 /** The maximum allowed number of signature check operations in a block (network rule) */
 static const int64_t MAX_BLOCK_SIGOPS_COST = 80000;
+/** Smallest possible transaction size */
+static const unsigned int MIN_TRANSACTION_SIZE = 60;
```

Comment:
Maybe add a comment to explain that this is not an independent consensus rule, but just the shortest otherwise valid serialized transaction?